src/OnboardingSPA/pages/Steps/BasicInfo/stylesheet.scss

```diff
@@ -3,6 +3,12 @@
     align-items:center;
     flex-direction: column;
     justify-content: center;
+    animation: fadeIn 500ms ease-in-out forwards;
```

Comment:
Can we get rid of all this CSS as we won't be needing it right?

Reply:
Yes, removing it